docs/sources/dashboards/assess-dashboard-usage/index.md

```diff
@@ -106,3 +106,7 @@ You can sort the dashboards by:
 - Views 30 days
 
 {{< figure src="/static/img/docs/enterprise/improved-search-7-5.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Usage insights dashboards
+
+If you set up your installation to [export logs of usage insights]({{< relref "../../setup-grafana/configure-security/export-logs/" >}}), we've created two dashboards to help you take advantage of this data. First is an [overview dashboard](https://grafana.com/grafana/dashboards/13785-usage-insights/) to provide a top-level perspective of user activity. Second is a [data source details](https://grafana.com/grafana/dashboards/13786-grafana-usage-insights-datasource-details/) dashboard to provide a view of data source activity and health. You can click the previous links to download the respective dashboard JSON, then import into your Grafana installation.
```

Comment:
I'm not sure I'm the best person to review, because I have some open questions still:

1. Do we actually intend for Usage Insights to be available to OSS customers? Or do we want this really to be a paid feature, and an incentive to upgrade?

Reply:
NIT: Instead of "First" and "second", maybe a numbered-list would be simpler here?